atriaapp/atriacalendar/models.py

```diff
@@ -119,8 +129,62 @@ def __str__(self):
         return self.title + ", " + self.location
 
 
-# Base class for organizations that use the Atria platform
-class AtriaOrganization(models.Model):
-    org_name = models.CharField(max_length=40)
-    date_joined = models.DateTimeField(default=timezone.now)
+# general announcements from an Org (will show up on a user's feed if they have an org relationship)
+class AtriaOrgAnnouncement(models.Model):
+    org_id = models.ForeignKey(AtriaOrganization, on_delete=models.CASCADE)
+    added_by_user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    title = models.CharField(max_length=80)
+    content = models.CharField(max_length=4000)
+    date_added = models.DateTimeField(default=timezone.now)
+    effective_date = models.DateTimeField(default=timezone.now)
+    end_date = models.DateTimeField(blank=True)
+
+
+# type of user/org relationship, type can be:
+#   Member = user is a regular attendee and/or volunteer
+#   Staff = user works at org and can maintain org information
+#   Admin = user can manage org/user relationships for this org
+class RelationType(models.Model):
+    relation_type = models.CharField(max_length=20)
+    relation_description = models.CharField(max_length=80)
+
+
+# Association class for user/organization relationship
+class AtriaRelationship(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    org_id = models.ForeignKey(AtriaOrganization, on_delete=models.CASCADE)
+    relation_type = models.ForeignKey(RelationType, verbose_name='relationship', on_delete=models.CASCADE)
     status = models.CharField(max_length=8)
+    effective_date = models.DateTimeField(default=timezone.now)
+    end_date = models.DateTimeField(blank=True)
+
+
+# type of user/event relationship:
+#   Bookmark
+#   Attend
+#   Volunteer
+#   Organize (for org staff only)
+class EventAttendanceType(models.Model):
+    attendance_type = models.CharField(max_length=20)
+    attendance_description = models.CharField(max_length=80)
+
+
+# set of "bookmarked" events (for a user)
+class AtriaBookmark(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    event_id = models.ForeignKey(AtriaEvent, on_delete=models.CASCADE)
+    bookmark_type = models.ForeignKey(EventAttendanceType, verbose_name='bookmark_type', on_delete=models.CASCADE)
```

Comment:
`verbose_name` being set here _undoes_ Django's internal conversion of `bookmark_type => 'Bookmark Type'. Is this intended?